src/users/UserEdit.tsx

```diff
@@ -0,0 +1,30 @@
+import {
+  Edit,
+  SimpleForm,
+  TextInput,
+  ReferenceInput,
+  AutocompleteInput,
+  email,
+} from "react-admin";
+
+const filterToQuery = (searchText: any) => ({
+  "name@ilike": `%${searchText}%`,
+});
+
+export const UserEdit = () => (
+  <Edit resource="users" redirect="/users_view">
```

Comment:
You should also add `mutationOptions` to provide a `onSuccess` with a `refresh()`

https://marmelab.com/react-admin/Edit.html#mutationoptions